include/session/config/base.hpp

```diff
@@ -188,10 +267,14 @@ class ConfigBase : public ConfigSig {
     // deleted at the next push.
     void set_state(ConfigState s);
 
-    // Invokes the `logger` callback if set, does nothing if there is no logger.
-    void log(LogLevel lvl, std::string msg) {
+    // Invokes the `logger` callback if set, does nothing if there is no logger.  Arguments go
+    // through fmt::format.
+    //
+    // TODO: replace used of this with oxen-logging!
```

Comment:
I think this has been replaced in the `libQuic` branch (not that that changes anything, but just an FYI)

Reply:
Yeah I thought we had it in *some* branch, just wasn't sure where.  There might be a bit of minor merge conflict around this but hopefully not too bad.